packages/ag-charts-community/src/chart/chart.ts

```diff
@@ -461,7 +462,9 @@ export abstract class Chart extends Observable implements ModuleInstance, ChartS
             const seriesDatumIndex = s.datumIndexForCategoryValue(categoryValue);
             const seriesTooltipContent =
                 seriesDatumIndex == null ? undefined : s.getTooltipContent(seriesDatumIndex, undefined);
-            return seriesTooltipContent == null ? [] : [seriesTooltipContent];
+            // Exclude series with all missing data from shared tooltips
+            if (seriesTooltipContent == null || hasAllMissingData(seriesTooltipContent)) return [];
```

Comment:
I think this should be handled solely in tooltip content rather than having this logic spread out

Reply:
Good shout - done